lib/sequence_diagram/sequencer.rb

```diff
@@ -0,0 +1,72 @@
+require "ostruct"
+
+module SequenceDiagram
+  class Sequencer
+    def self.sequence
```

Comment:
`call` also works, but I have worried that it might have baggage attached to it based on how it works in native ruby features.